17.4.12.13.44

3442853561 opened the pull request.

akihikodaki commented:
By the way, I found your proposal in this tweet of `@rustlang`.
https://twitter.com/rustlang/status/808490052301049856
I am glad to see it. Good luck!

3442853561 commented:
Sorry, I forgot to check out the Pull requests. Resulting in some version of the problem. I have copied your changes. Thank you for your support.

akihikodaki commented:
I have reviewed your commits and confirmed the essential changes are merged. I'll close this issue.

Commit:
Change example.rs to a simpler and more welcoming one

Commit:
Merge branch 'master' into feat/simpler_welcoming_example

Commit:
Make example use an array and a for loop

brson committed:
Merge pull request #756 from nubark/feat/simpler_welcoming_example

Change example.rs to a simpler and more welcoming one

brson committed:
Make example even better

brson committed:
Merge pull request #774 from brson/ex

Make example even better

brson committed:
Adjust code box to fit new example

malbarbo committed:
Android is now supported

brson committed:
Example cleanup

brson committed:
Merge pull request #776 from brson/ex

Example cleanup

brson committed:
Merge pull request #775 from malbarbo/android

Android is now supported

brson committed:
Fix spaces in example

brson committed:
Merge pull request #777 from brson/ex

Fix spaces in example

brson committed:
Add GiGa infosystems to friends

brson committed:
Add Linkerd to friends

alexcrichton committed:
Merge pull request #778 from brson/friends

Add GiGa infosystems and Linkerd to friends

eyeplum committed:
Refinement wording for Chinese example at index

alexcrichton committed:
Merge pull request #779 from eyeplum/master

Refinement wording for Chinese example at index page

vasily-kirichenko committed:
Update index.html

Better wording. "безумно" can be translated as "crazy" and sounds non-professionally. "невероятно" can be translated as "incredibly", which is sounds more formally and appropriate for a front page.

alexcrichton committed:
Merge pull request #780 from vasily-kirichenko/patch-1

Update index.html

HadrienG2 committed:
Fix website formatting issue

The Markdown engine used for rendering the rust-lang website does not like headings which are not preceded by a linefeed, i.e. this code...

```
Something
## A heading
```

...will be badly rendered as `Something ## A heading`.

Note that this behaviour violates the CommonMark spec ( http://spec.commonmark.org/0.27/#atx-headings : "ATX headings need not be separated from surrounding content by blank lines, and they can interrupt paragraphs"). So if you know more than me about the rendering engine in use, you can ask them to fix their parser.

But since putting blank lines before and after headings makes the markdown style more consistent/readable anyway, I've went for the alternate option of working around the parser bug :)

steveklabnik committed:
Merge pull request #781 from HadrienG2/patch-1

Fix website formatting issue

steveklabnik committed:
Core team changes

+ @carols10cents, @nrc
- @pcwalton
